client/me/index.js

```diff
@@ -65,6 +65,13 @@ export default function() {
 			controller.purchases.noSitesMessage,
 			controller.purchases.list
 		);
+
+		page(
+			paths.purchases.listNotice(),
+			controller.sidebar,
+			controller.purchases.noSitesMessage,
```

Comment:
I always forget about it, we have sites always loaded when dealing with component loaded by `page` helper :+1: